source/frontend/src/features/properties/map/project/update/UpdateProjectContainer.tsx

```diff
@@ -0,0 +1,81 @@
+import { Input, Select, SelectOption, TextArea } from 'components/common/form';
+import { Section } from 'features/mapSideBar/tabs/Section';
+import { SectionField } from 'features/mapSideBar/tabs/SectionField';
+import { Formik, FormikHelpers, FormikProps } from 'formik';
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import styled from 'styled-components';
+
+import { ProjectForm } from '../add/models';
+
+export interface IUpdateProjectContainerProps {
+  initialValues: ProjectForm;
+  projectStatusOptions: SelectOption[];
+  projectRegionOptions: SelectOption[];
+  validationSchema?: any | (() => any);
+  onSubmit: (values: ProjectForm, formikHelpers: FormikHelpers<ProjectForm>) => void | Promise<any>;
+}
+
+const UpdateProjectContainer = React.forwardRef<FormikProps<any>, IUpdateProjectContainerProps>(
```

Comment:
This doesn't seem relevant to this PR but I guess is fine to keep it since you will working with this in your next story/PR